packages/compiler/test/parser.test.ts

```diff
@@ -105,6 +107,9 @@ describe("compiler: syntax", () => {
         [/Cannot use default with non optional properties/],
       ],
       ["model", [/Identifier expected/]],
+      ["model Car is vehicle", [/'{' expected/]],
```

Comment:
should this be `;` or `{` expected?

Reply:
hmm, I suppose so...

Reply:
Not that concerned if that's too much trobule.